components/resourceDetails/index.jsx

```diff
@@ -72,6 +77,22 @@ export default function ResourceDetails({
   const { accessControl } = useContext(AccessControlContext);
   const { enabled } = useContext(FeatureContext);
   const useNewAcpUi = enabled(NEW_ACP_UI_ENABLED);
+  const [actionsAccordion, setActionsAccordion] = useLocalStorage(
+    getAccordionKey(dataset, "actions"),
+    true
+  );
+  const [detailsAccordion, setDetailsAccordion] = useLocalStorage(
+    getAccordionKey(dataset, "details"),
+    false
+  );
+  const [permissionsAccordion, setPermissionsAccordion] = useLocalStorage(
+    getAccordionKey(dataset, "permissions"),
+    false
+  );
+  const [sharingAccordion, setSharingAccordion] = useLocalStorage(
```

Comment:
I prefer the idea of using querystring params instead of localstorage - let's revisit this later.